.travis.yml

```diff
@@ -40,5 +41,26 @@ script:
   - web-push-testing-service start example -p 9012
   - composer test:unit
   - web-push-testing-service stop example
-  - composer test:typing
-  - composer test:syntax
+
+jobs:
+    allow_failures:
+        - php: nightly
+
+    include:
+        - stage: Metrics and quality
+          env: STATIC_ANALYSIS
```

Comment:
Looks like this fails due to the new function not  being available in lower PHP versions